app/code/Magento/PageCache/Test/Mftf/ActionGroup/AssertValueIsNotCachedActionGroup.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertValueIsNotCachedActionGroup">
```

Comment:
Incorrect AG name format
Should be: {Assert}{Admin|Storefront}{name}{ActionGroup}
{name} also should be descriptive enough, this one too general

Reply:
done